test/test_tagging.py

```diff
@@ -0,0 +1,230 @@
+import json
+from typing import (
+    Any,
+    Hashable,
+    Optional,
+    Tuple,
+)
+import unittest
+from unittest.mock import (
+    MagicMock,
+    patch,
+)
+
+from azul.deployment import (
+    populate_tags,
+)
+from azul.logging import (
+    configure_test_logging,
+)
+from azul_test_case import (
+    AzulUnitTestCase,
+)
+
+
+def setupModule():
+    configure_test_logging()
+
+
+class TestTerraformResourceTags(AzulUnitTestCase):
+
+    def assertDictEqualPermissive(self, expected: Any, actual: Any):
```

Comment:
You're correct, I mistakenly applied the hints to permissive_compare to this method.

In this case, should TestTerraformResourceTags.permissive_compare retain explicit `Any` hints, or should I omit them since it is implied?

Reply:
My original comment already answers that:

> The argument types should be more restrictive.